examples-next/graphql-api-endpoint/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# keystone-next-app
+# keystone-app
```

Comment:
I don't think we want to change this package.

Reply:
Reverted.